data/categories.txt

```diff
@@ -346,7 +346,7 @@ cs:2Bar|pivo|pití|jídlo|voda|pitná voda|tekutiny
 sk:2Bar|pivo|pitie|jedlo|voda|pitná voda|tekutiny
 nl:2Bar|2pub|2café|bier|drinken|eten
 zh-Hant:1酒吧|酒館|飲酒|PUB|吧台|飲食
-pl:2Bar|2pub|piwo|napoje|jedzenie
+pl:2Bar|alkohole|drinki|napoje|jedzenie
```

Comment:
All words except the first one are shown nowhere in the app: they are used only for searching. So when you remove the "pub" synonym from the bar type, user's will have to type both words into a search field to find a place to drink beer.